### PR TITLE
`browser.Dockerfile`: Fix `rm` command

### LINKED ISSUE #309: Renamed directory paths in `browser.Dockerfile`
I am creating a package for deployment using `browser.Dockerfile`. I noticed that some paths mentioned here:

https://github.com/eclipse-theia/theia-blueprint/blob/c70b99f6fed23cf6ee4ca9105b93c761f808630b/browser.Dockerfile#L26

do not exist. `theia-blueprint-launcher` and `theia-blueprint-updater` seem to be renamed to `launcher` and `updater`. If true, I can create a PR.

### PR DESCRIPTION
**Pull request by goekce**
#### What it does
Fixes wrong paths and removes an unnecessary option.

#### How to test
Rebuilding the Docker image

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

I don't know what *thoroughly* means however I could successfully build the Docker image.

---
closes  #309

### CODE REVIEW

**Comment by marcdumais-work** [comment]
Hi @goekce,

Thanks for this PR.

> [remove -f because the listed directories will always exist](https://github.com/eclipse-theia/theia-blueprint/pull/311/commits/8b7c151a07b273a5bc90804a4eec72b859883acf) 
\
This may help to catch future errors

I am not sure about this part. When I test locally on my laptop (not in Docker), running `rm -f <non-existing directory>`, a success status (0) is still returned. Also, I'm always a bit worries about permission issues with Docker, when there's a cross-over between host and container permissions, like in this case. Would you mind removing that commit?

**Comment by goekce** [comment]
> `rm -f <non-existing directory>`, a success status (0) is still returned
I believe there is a misunderstanding Marc. `rm -f` always succeeds and `rm` will output an error if a directory/file does not exist. This way you can catch future errors.

> when there's a cross-over between host and container permissions

I don't understand this point, can you rephrase?

**Comment by marcdumais-work** [comment]
@goekce sorry - I made a mistake trying to test this locally during the weekend. You are right, `rm -f` will end with an error status if one or more of the folders do not exist, making it possible to catch a potential future name mismatch re-occurrence.

>> when there's a cross-over between host and container permissions

> I don't understand this point, can you rephrase?

It was out of an abundance of caution, but it's easy to introduce file ownership permission issues when bringing host files into a container. e.g. if the `uid` of the user on the host is different than in the container. Then, depending on the `umask` the files had on the host, a different user/`uid` might be prompted before the files/folders can be removed, if rm is used without "f" (or they could be impossible to remove altogether without running `chmod` first. However in this case here, I believe the folders are removed by user `root`, and so it should not matter :)

**Review by marcdumais-work — APPROVED** [review]
LGTM, thanks @goekce for this contribution!
